### PR TITLE
milestone 3

### PR DESCRIPTION
**Pull request by carloshdelreal**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,14 +1,18 @@
 Rails.application.routes.draw do
+  get 'posts/new'
```
Comment (mosaaleb) [inline config/routes.rb]: - [ ] There are two ways for defining routes [Resourceful routes](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) or [Non-resourceful routes](https://guides.rubyonrails.org/routing.html#non-resourceful-routes). Let's choose only one of them for posts routes.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @post = current_user.posts.build(post_params)
+    if @post.save
+      flash[:success] = 'Your post has been created'
+      redirect_to posts_index_path
+    else
+      render 'new'
+    end
+  end
+
+  def index
```
Comment (mosaaleb) [inline app/controllers/posts_controller.rb]: - [ ] Index displays posts in a timeline (recent to older). We can achieve this by using [default scope](https://apidock.com/rails/ActiveRecord/Base/default_scope/class) in `post.rb`. An example could be [this](https://stackoverflow.com/questions/3393687/default-sort-order-for-a-rails-model).

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def new
```
Comment (mosaaleb) [inline app/controllers/posts_controller.rb]: - [ ] Let's add posts edit and delete features.

File: config/routes.rb (hunk 1 above)
Comment (trox115) [reply config/routes.rb]: Always have asked what the resources do. Thank you. Corrected

**Pull request merged**